generator/src/lib.rs

```diff
@@ -2128,10 +2128,18 @@ pub fn generate_struct(
             #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VkAccelerationStructureInstanceKHR.html>"]
             pub struct AccelerationStructureInstanceKHR {
                 pub transform: TransformMatrixKHR,
+                /// Use [`pack_24_8(instance_custom_index, mask)`][Self::pack_24_8()] to construct this field
                 pub instance_custom_index_and_mask: u32,
+                /// Use [`pack_24_8(instance_shader_binding_table_record_offset, flags)`][Self::pack_24_8()] to construct this field
                 pub instance_shader_binding_table_record_offset_and_flags: u32,
                 pub acceleration_structure_reference: AccelerationStructureReferenceKHR,
             }
+            impl AccelerationStructureInstanceKHR {
+                pub fn pack_24_8(low_24: u32, high_8: u8) -> u32 {
```

Comment:
We might want a comment here like 'Pack `low_24` and `high_8` into the same `u32` such that `high_8` occupies the highest 8 bits and `low_24` occupies the lowest 24 bits' or something.

Otherwise, this looks great!